commcare_connect/opportunity/views.py

```diff
@@ -152,8 +152,11 @@ def form_valid(self, form):
         opportunity = form.instance
         opportunity.modified_by = self.request.user.email
         users = form.cleaned_data["users"]
-        if users:
-            add_connect_users.delay(users, form.instance.id)
+        filter_country = form.cleaned_data["filter_country"]
+        filter_credential = form.cleaned_data["filter_credential"]
```

Comment:
Do these get validated per https://github.com/dimagi/commcare-connect/pull/323/files#diff-5eeea0bc14a923bfbb6d28553079e296782035b7dae74aa86fa1f0f644355135R91

Reply:
Yes, we only offer a choice on the select and do not let the user enter custom data.